src/gedcom.ts

```diff
@@ -118,16 +133,21 @@ function createEvent(entry: GedcomEntry | undefined): JsonEvent | undefined {
   if (!entry) {
     return undefined;
   }
+  const typeTag = findTag(entry.tree, 'TYPE');
   const dateTag = findTag(entry.tree, 'DATE');
-  const date = dateTag && dateTag.data && getDate(dateTag.data);
   const placeTag = findTag(entry.tree, 'PLAC');
+  const notes = createNotes(findTag(entry.tree, 'NOTE'));
+
+  const date = dateTag && dateTag.data && getDate(dateTag.data);
   const place = placeTag && placeTag.data;
   if (date || place) {
     const result: JsonEvent = date || {};
     if (place) {
       result.place = place;
     }
     result.confirmed = true;
+    result.type = typeTag ? typeTag!.data : undefined;
+    result.notes = notes;
```

Comment:
```typescript
result.notes = createNotes(findTag(entry.tree, 'NOTE'));
```

Reply:
original intent was to have a  single block of initialization. but as you prefer